README.md

```diff
@@ -112,47 +114,103 @@ trajectoryTimestamped:1:1:1627551892437;2:2:1627551892448;3:3:1627551892459|xy
 By default, teleplot will display all the incoming telemetry as a chart, while this is handy for new user with small amount of data, this might not be desired with lots of data.
 The `np` (for no-plot) flag can be used to prevent this behavior:
 - `myValue:1627551892437:1234|np`
-- `trajectory:12.3:45.67|xynp`
+- `trajectory:12.3:45.67|xy,np`
 
 ### Publishing 3D telemetries
 
-To send 3D shapes to teleplot, use this syntax : `3D|A:B|E`, where
-**A** is the timestamp and is **optional**
-**B** is a json object representing the shape
-**E** is containing flags
+To send 3D shapes to teleplot, use this syntax : `3D|A:B:C|E`, where
+**A** is the name of the shape telemetry
+**B** is the timestamp of the telemetry and is **optional**
+**C** is a text representing the shape
+**E** is containing flags and is **optional**
 
-for the moment, there are only two shapes available, cubes and spheres.
+### Writing **A** (the name of the shape telemetry)
 
-For a sphere, **B** should be as follow : 
-{
-    "shape" : "sphere",
-    "position" : {"x": _ , "y": _ , "z" : _ },
-    "rotation" : {"x": _ , "y": _ , "z" : _ },
-    "precision" : _ ,
-    "radius" : _ ,
-}
+if **A** contains a comma, the text after the comma will be considered as a 'widget label', if multiple telemtries are sent with the same
+widget label, they will automatically be displayed on the same widget.
+The text before the comma will be considered as the name of the shape telemetry.
 
-For a cube, **B** should be as follow : 
-{
-    "shape" : "cube",
-    "position" : {"x": _ , "y": _ , "z" : _ },
-    "rotation" : {"x": _ , "y": _ , "z" : _ },
-    "height" : _ ,
-    "width" : _ ,
-    "depth" : _ ,
-}
+If **A** doesn't contain a comma, its whole text will be considered as the name of the shape telemetry.
+
+#### Writing **C** (the text representing the shape)
+
+**C** is built as a concatenation of the shape properties followed by their values, the whole with colons in between.
+
+LIST OF PROPERTIES : 
+
+- "shape" or "S"  => the shape type (either "cube" or "sphere" for the moment).
+
+- "position" or "P" => the position of the center of the sphere in a cartesian coordinate system 
+    1st argument : x, 2nd argument : y, 3rd argument : z
```

Comment:
Doing a small test of a moving sphere, it seems that the y axis is the height while I would have expected Z to be the height:
![image](https://user-images.githubusercontent.com/9192862/198554238-b7b7ddd7-2dbb-454a-8756-d032068e8cd3.png)